InteractionBase/CMakeLists.txt

```diff
@@ -192,4 +192,4 @@ add_library(InteractionBase SHARED
 	src/handlers/HMovableItem.cpp
 )
 
-envision_plugin(InteractionBase VisualizationBase FilePersistence)
\ No newline at end of file
+envision_plugin(InteractionBase VisualizationBase FilePersistence VersionControlUI)
```

Comment:
The correct thing to do is to move the `CDiff` command in this plug-in. So please do that.

Regardless of what is the correct resolution of this specific problem, please be more meticulous in the future. You should always think carefully what you're building and what dependencies there are. Introducing a circular dependency should never happen.